### ISSUE TITLE
Suggestion: Procedual Generated Alchemy Circles

### ISSUE THREAD

**Issue by Unrepentant-Atheist** [WayofTime/BloodMagic#1271]
https://github.com/CiaccoDavide/Alchemy-Circles-Generator

'nuff said :P

**Comment by WayofTime** [comment]
While it will certainly help with ideas of structures, I try to design the arrays from the ground up to match each array's purpose.

**Comment by Unrepentant-Atheist** [comment]
Yeah, sure, make sense. I just thought it neat and post it here. If it helps, it does, if not, then not. It's open source after all :D
